src/amo/components/AddonReviewCard/index.js

```diff
@@ -317,6 +318,14 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
           </span>
         );
       }
+      byLine = (
+        <Link
```

Comment:
I think it would look better if the links were underlined in `$grey-50` -- they will fit with the header style better. It will be more of an issue when moving the link to the timestamp.

<img width="822" alt="screenshot 2018-09-14 12 38 21" src="https://user-images.githubusercontent.com/55398/45566063-2d64ab00-b81b-11e8-8f23-ea14637055c9.png">